llvm/test/MC/CAS/linker-options.ll

```diff
@@ -0,0 +1,7 @@
+; RUN: mkdir -p %t.dir/cas
+; RUN: llc -O0 -mtriple=x86_64-apple-darwin -filetype=obj --cas %t.dir/cas -cas-backend -o - %s | llvm-readobj --macho-linker-options - > %t
```

Comment:
Add the FileCheck to make sure you are actually generating the correct output?

Maybe also REQUIRES x86_64 backend?